pkg/kudoctl/packages/install/package.go

```diff
@@ -73,12 +73,13 @@ func Package(
 		return nil
 	}
 
-	if err := install.Instance(client, resources.Instance); err != nil {
+	installed, err := install.Instance(client, resources.Instance)
```

Comment:
hmm why is this better than what was here before? 🤔

Reply:
I'm ok with this, tbh, it enforces the "always use the returned object if you create one" pattern... It allows you to update the object later one:

```
pod := corev1.Pod{ ... }
pod := client.Create(pod)
pod.metadata.Labels["something"] = "addedlabel"
pod := client.Update(pod)
```

This won't work if you try to do:
```
pod := corev1.Pod{ ... }
client.Create(pod)
pod.metadata.Labels["something"] = "addedlabel"
client.Update(pod)
```

because of the added fields from the api-server...

Reply:
see my [comment above](https://github.com/kudobuilder/kudo/pull/1636#discussion_r466526744)